src/process.cpp

```diff
@@ -128,27 +123,61 @@ int proc_t::execute(int app_id) {
     terminate();
   });
 
-  for(; _undo_it != std::end(proc.prep_cmds); ++_undo_it) {
-    auto &cmd = _undo_it->do_cmd;
+  // Execute global prep commands if enabled
+  if (_app.include_global_prep)
```

Comment:
What was the reasoning for an additional for loop in this case? Why not just create a new iterator by merging them together? You can decide to just not add it to the new iterator if the checkmark is false.

Reply:
I actually had them combined previously, before adding the ability to exclude them per app, but split them to make things easier to debug when adding that feature in. There's really no reason they couldn't be combined though, so I may change it back now that you've pointed it out.

Reply:
I was just thinking when someone adds the capability to reference moonlight client variables (like client resolution, framerate, hdr setting, etc) it would be easier to implement if it's using an iterator. Multiple loops could work as well if you refactor it to use a function to do most of the work too

Reply:
I've updated it to load everything into a single vector for each app, so only a single loop is needed now.